misk/src/test/kotlin/misk/perf/PauseDetectorTest.kt

```diff
@@ -0,0 +1,128 @@
+package misk.perf
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.spi.ILoggingEvent
+import com.google.common.base.Ticker
+import misk.ServiceManagerModule
+import misk.concurrent.FakeTicker
+import misk.concurrent.Sleeper
+import misk.inject.KAbstractModule
+import misk.logging.LogCollectorModule
+import misk.metrics.v2.FakeMetricsModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import wisp.logging.LogCollector
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+@MiskTest(startService = true) // NB: only starting services here to get log collection to work.
```

Comment:
What does NB mean? For a longer comment line maybe put above the annotation

Reply:
Nota bene (linked in another comment). Done